site/src/pages/WorkspacePage/WorkspacePage.test.tsx

```diff
@@ -171,7 +174,7 @@ describe("WorkspacePage", () => {
     await testStatus(MockDeletingWorkspace, DisplayStatusLanguage.deleting)
   })
   it("shows the Deleted status when the workspace is deleted", async () => {
-    await testStatus(MockDeletedWorkspace, DisplayStatusLanguage.deleted)
+    await testStatus(MockDeletedWorkspace, t("workspace_status.deleted", { ns: "common" }))
```

Comment:
We should use our new translations in our tests, too, so our tests don't become brittle.